Source/missiles.cpp

```diff
@@ -1396,7 +1396,7 @@ void AddSpectralArrow(Missile &missile, AddMissileParameter &parameter)
 	int av = 0;
 
 	if (missile.sourceType() == MissileSource::Player) {
-		const Player &player = *missile.sourcePlayer();
+		Player &player = *missile.sourcePlayer();

```

Comment:
Why remove the `const`? I don't think `player` is modified her?

Reply:
Can't iterate through equipment if it's const

Reply:
Would it be better to use a traditional for loop to retain player as const?

Reply:
Ah, right, this will be easy to fix once we have C++17 but for now non-const is fine.

Reply:
I'll add a comment by it so we know to readd const at a later time